### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by villander**
- [x] Update

- [x] Add code of conduct ( Ember community )

 README with new engine status and new module API

https://discuss.emberjs.com/t/readers-questions-what-is-the-status-of-ember-engines/14236

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -527,18 +534,16 @@ An application that contains this engine must explicitly fulfill these
...
-Ember.MODEL_FACTORY_INJECTIONS = true;
-
 const { modulePrefix, podModulePrefix } = config;
 
-const App = Ember.Application.extend({
-  modulePrefix,
-  podModulePrefix,
+const App = Application.extend({
+  modulePrefix: config.modulePrefix,
+  podModulePrefix: config.podModulePrefix,
```
Comment (mattmarcum) [inline README.md]: because of  line 542 you don't need to change these two preceding lines

File: README.md [hunk 2]
```diff
@@ -553,7 +558,7 @@ const App = Ember.Application.extend({
   }
 });
 
-loadInitializers(App, modulePrefix);
+loadInitializers(App, config.modulePrefix);
```
Comment (mattmarcum) [inline README.md]: same here, `config.` isn't needed

File: README.md (hunk 1 above)
Comment (villander) [reply README.md]: You are wrong. This is the actual blueprint of Ember's latest versions without the use of destructuring, and the new modules API. 

https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/app/app.js

File: README.md (hunk 2 above)
Comment (villander) [reply README.md]: Same thing here

File: README.md (hunk 1 above)
Comment (villander) [reply README.md]: Thanks, i fixed

File: README.md (hunk 2 above)
Comment (villander) [reply README.md]: Thanks, i fixed

**Comment by villander** [comment]
@stefanpenner we can do merge this?

**Comment by villander** [comment]
I'll enjoy it, and sync with that PR on the website ok? When it's okay, I'll let you know.

https://github.com/ember-engines/ember-engines.com/pull/37

**Pull request merged**